FakeXrmEasy.Tests.Shared/FakeContextTests/RetrieveMultiple/RetrieveMultipleTests.cs

```diff
@@ -566,5 +567,26 @@ public void Should_Allow_Using_Aliases_with_Dot()
             Assert.True(accounts.Entities.First().Contains("primary.contact.firstname"));
             Assert.Equal("Jordi", accounts.Entities.First().GetAttributeValue<AliasedValue>("primary.contact.firstname").Value);
         }
+
+        [Fact]
+        public void TheCorrectResultIsReturnedWhenUsingConditionOperatorInWithGuid()
+        {
+            var context = new XrmFakedContext();
+            context.ProxyTypesAssembly = Assembly.GetExecutingAssembly();
+            var contact = new Crm.Contact()
+            {
+                Id = Guid.NewGuid()
+            };
+            context.Initialize(contact);
```

Comment:
Minor feedback @BetimBeja , when using Initialize() with early bound there's no need to specify the ProxyTypesAssembly manually, FakeXrmEasy will retrieve internally

Reply:
Thanks again for this PR, been hectic after christmas but will spend more time on this again :)

Reply:
I removed the unnecessary line of code.
I am always glad to help with this 😃